src/Tools/AI Test Toolkit/src/TestSuite/AITTestMethodLine.Table.al

```diff
@@ -166,6 +166,18 @@ table 149032 "AIT Test Method Line"
             FieldClass = FlowField;
             CalcFormula = count("AIT Log Entry" where("Test Suite Code" = field("Test Suite Code"), "Test Method Line No." = field("Line No."), Version = field("Base Version Filter")));
         }
+        field(32; Iterations; Integer)
```

Comment:
IMO Iteration should be on the suite level. We can also have it on the line, but then it is following the action on the line itself.